src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -2933,7 +2933,7 @@ class DropTriggerStatementSegment(BaseSegment):
         Ref("TriggerReferenceSegment"),
         "ON",
         Ref("TableReferenceSegment"),
-        OneOf("CASCADE", "RESTRICT", optional=True),
+        Ref("DropBehaviorGrammar", optional=True),
```

Comment:
update `type` to `type = "drop_trigger_statement"`

Reply:
√